src/entity/posts/utils.ts

```diff
@@ -340,6 +341,20 @@ export const createSharePost = async (
       },
     });
 
+    const vordrStatus = await checkWithVordr(
+      { post: createdPost },
+      { con, userId, req: ctx.req },
+    );
+
+    if (vordrStatus === true) {
+      createdPost.banned = true;
+      createdPost.showOnFeed = false;
```

Comment:
I think we also have flags.showOnFeed and flags.banned?

Reply:
Also set flags now